src/workers/parser/getSymbols.js

```diff
@@ -199,7 +200,16 @@ function extractSymbol(path: SimplePath, symbols) {
       return;
     }
 
-    if (t.isProperty(path.parent)) {
+    const inAssignment = t.isAssignmentPattern(path.getAncestor(3));
+    // explain what this is
+    const isFunctionParamShorthand =
```

Comment:
Hey sunny, can you explain what each of these variables is checking for? It'd be nice to have a comment next to each of them that explains their purpose and perhaps an example of something they'd catch